pkgs/os-specific/linux/igh-ethercat-master/default.nix

```diff
@@ -0,0 +1,59 @@
+{ lib, stdenv, fetchFromGitLab, autoconf, automake, pkg-config, libtool, kmod, iproute2, kernel ? null }:
```

Comment:
kernel shouldn't be null if this is a kernel module or does this build something else, too?

Reply:
That should not be there, it does build some binaries that is used as a front-end for the bus, but they seem to need the kernel headers when running configure.